pkg/stub/handler.go

```diff
@@ -1763,101 +1786,96 @@ func (g *DefaultResourceFileLister) List(dir string) (files []os.FileInfo, err e
 }
 
 type InClusterInitter interface {
-	init()
+	init(h *Handler, restconfig *restclient.Config)
 }
 
 type defaultInClusterInitter struct {
-	h *Handler
 }
 
-func (g *defaultInClusterInitter) init() {
-	restconfig, err := getRestConfig()
-	if err != nil {
-		logrus.Errorf("failed to get rest config : %v", err)
-		panic(err)
-	}
-	g.h.restconfig = restconfig
+func (g *defaultInClusterInitter) init(h *Handler, restconfig *restclient.Config) {
+	h.restconfig = restconfig
 	tempclient, err := getTemplateClient(restconfig)
 	if err != nil {
 		logrus.Errorf("failed to get template client : %v", err)
 		panic(err)
 	}
-	g.h.tempclient = tempclient
+	h.tempclient = tempclient
 	logrus.Printf("template client %#v", tempclient)
 	imageclient, err := getImageClient(restconfig)
 	if err != nil {
 		logrus.Errorf("failed to get image client : %v", err)
 		panic(err)
 	}
-	g.h.imageclient = imageclient
+	h.imageclient = imageclient
 	logrus.Printf("image client %#v", imageclient)
 	coreclient, err := corev1client.NewForConfig(restconfig)
 	if err != nil {
 		logrus.Errorf("failed to get core client : %v", err)
 		panic(err)
 	}
-	g.h.coreclient = coreclient
+	h.coreclient = coreclient
 	configclient, err := configv1client.NewForConfig(restconfig)
 	if err != nil {
 		logrus.Errorf("failed to get config client : %v", err)
 		panic(err)
 	}
-	g.h.configclient = configclient
+	h.configclient = configclient
 }
 
-type SDKWrapper interface {
+type CRDWrapper interface {
 	Update(samplesResource *v1alpha1.SamplesResource) (err error)
 	Create(samplesResource *v1alpha1.SamplesResource) (err error)
 	Get(name string) (*v1alpha1.SamplesResource, error)
 }
 
-type defaultSDKWrapper struct {
+type generatedCRDWrapper struct {
 	h *Handler
+	c sampleclientv1alpha1.SamplesResourceInterface
```

Comment:
s/c/client/ ?  (I'm not a fan of single letter variable names unless it's a temp/iterator var)

Reply:
part of next push